edge-functions/power-parity-pricing-strategies/pages/_app.tsx

```diff
@@ -1,10 +1,12 @@
-import Image from 'next/image'
 import type { AppProps } from 'next/app'
 import type { LayoutProps } from '@vercel/examples-ui/layout'
+import Image from 'next/image'
 import { getLayout, Text } from '@vercel/examples-ui'
-import '@vercel/examples-ui/globals.css'
+
 import map from '../public/map.svg'
 
+import '@vercel/examples-ui/globals.css'
+
```

Comment:
@goncy What are these changes for?

Reply:
Just to have the types on top and the css on the bottom